indico/modules/rb/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -7,952 +7,330 @@
 
```

Comment:
Shouldn't this file also go inside the `edit/` subdir?

Reply:
I'm not sure, what do you think?
Since this is a `core` file, connected to the `RoomEditDetails` for me it made more sense to highlight it and move to the `rooms`.

Reply:
hm you could always re-export it from edit/index.js so you can `import RoomEditModal from './edit';` which I think is clearer (and that way anything for editing is in one place)

Reply:
I'm not sure if that is a better solution since the only thing I am importing and exporting is `RoomEditModal` itself, it is an unnecessary barrel. Nonetheless, I moved it into the `edit`.